<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_clculadorahipotecaria</class>
 <widget class="QDialog" name="Dialog_clculadorahipotecaria">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1099</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CALCULADORA HIPOTECARIA</string>
  </property>
  <widget class="QFrame" name="frame_historico">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1081</width>
     <height>691</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>4</number>
   </property>
   <widget class="QFrame" name="frame_mod_insert_ind">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>1051</width>
      <height>451</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <widget class="QRadioButton" name="rbtn1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Indice mensual</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbtn2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Int. legal anual</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_modinserind">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>INSERTAR REGISTRO :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_agnoind">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>A침o</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_mesind">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>60</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Mes</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_entind">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>60</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Entidades</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_euriborind">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>60</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Euribor</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_cajasind">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>60</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Cajas</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_bancosind">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>60</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Bancos</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_diferencialind">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>60</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Diferencial</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_fijoind">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>60</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Fijo</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_otroind">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>60</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Otro</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_modinsertent">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>80</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_modinsertbanc">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>80</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_modinsertcaja">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>80</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_modinserteuribor">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>80</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_modinsertfijo">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>80</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_modinsertdiferen">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>80</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_modinsertotro">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>80</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_entind_5">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>140</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Interes demora</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_fecfinaldinero_3">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>160</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_fecinicialdinero_3">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>160</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_agnoind_7">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>140</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>A침o</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_bancosind_5">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>140</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Fecha inicial</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_intdemoradinero_3">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>160</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_euriborind_5">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>140</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Observaci칩n</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_obsdinero_3">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>160</y>
       <width>211</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_intlegaldinero_3">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>160</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_mesind_5">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>140</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Interes legal</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_cajasind_5">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>140</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>fecha final</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_agnoind_9">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>390</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>A침o</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_fecfinaldinero_4">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>390</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>182</x>
       <y>310</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbtn3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Hist칩rico indices</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_modinsertotro_2">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>290</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_mesind_3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>310</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Mes:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>390</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_fecinicialdinero_4">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>390</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_entind_6">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>370</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Interes demora</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_cajasind_6">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>370</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>fecha final</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ejecutardinero_2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>280</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Cargar registro</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_cajasind_3">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>270</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Cajas</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_bancosind_3">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>270</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Bancos</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_intdemoradinero_4">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>390</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_otroind_2">
     <property name="geometry">
      <rect>
       <x>920</x>
       <y>270</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Otro</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_fijoind_2">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>270</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Fijo</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ejecutardinero_6">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>377</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Cargar registro</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_bancosind_6">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>370</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Fecha inicial</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_modinserteuribor_2">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>290</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbtn4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>390</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Hist칩rico Int. legal</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_modinsertdinero">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>261</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>MODIFICAR REGISTROS :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_diferencialind_2">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>270</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Diferencial</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_agnoind_3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>280</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>A침o:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_euriborind_3">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>270</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Euribor</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_modinsertent_2">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>290</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_obsdinero_4">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>390</y>
       <width>211</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_modinsertcaja_2">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>290</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_modinsertbanc_2">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>290</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_modinsertdiferen_2">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>290</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>280</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_mesind_6">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>370</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Interes legal</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_entind_3">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>270</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Entidades</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_intlegaldinero_4">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>390</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_modinsertfijo_2">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>290</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_euriborind_6">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>370</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Observaci칩n</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_4">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_5">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>80</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_6">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>160</y>
       <width>73</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_borrartablaeinsertar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>1041</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <widget class="QLabel" name="label_borrartablainsertar">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>401</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Borrar tabla e insertar nuevos datos desde fichero :</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbtn5">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>20</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Hist칩rico 칤ndices</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbtn6">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>20</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Hist칩rico interes dinero</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_limpiardatosingres">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>600</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Limpiar  campos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_delprograma">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>600</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SALIR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ejecutardinero_4">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>600</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Actualizar registro</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ejecutardinero_5">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>600</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Insertar registro</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_borrartablaeinsertar">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>600</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Importar archivo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_66">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>80</y>
      <width>231</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>VER TABLAS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plantilla">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Gu칤a plantilla importaci칩n</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
